### ISSUE TITLE
Failed to compile using incremental `plot(1:3)`

### ISSUE THREAD

**Issue by xiaodaigh** [JuliaLang/PackageCompiler.jl#207]
My toml file is

```julia
DataFrames
````

my `code.jl` is
```julia
using Plots
gr()
plot(1:3)
```

Then I run to compile them

```julia
using PackageCompiler
compile_incremental("toml.toml", "code.jl", force = true)
```

But I get the error

```julia
ERROR: LoadError: LoadError: error compiling _plot!: error compiling _display: could not load library "libGR.dll"
The specified module could not be found.

Stacktrace:
 [1] #plot#136(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::UnitRange{Int64}) at C:\Users\RTX2080\.julia\packages\Plots\QYETN\src\plot.jl:57
 [2] plot(::UnitRange{Int64}) at C:\Users\RTX2080\.julia\packages\Plots\QYETN\src\plot.jl:51
 [3] top-level scope at none:0
 [4] include at .\boot.jl:326 [inlined]
 [5] include_relative(::Module, ::String) at .\loading.jl:1038
 [6] include(::Module, ::String) at .\sysimg.jl:29
 [7] top-level scope at none:0
in expression starting at c:\git\ok2.jl:3
in expression starting at C:\Users\RTX2080\.julia\dev\PackageCompiler\sysimg\run_julia_code.jl:10
ERROR: failed process: Process(`'C:\Users\RTX2080\AppData\Local\Julia-1.1.0\bin\julia.exe' --optimize=3 '--output-o=C:\Users\RTX2080\.julia\dev\PackageCompiler\sysimg\sys.a' -g1 --track-allocation=none --startup-file=no --code-coverage=none --history-file=yes --inline=yes --math-mode=ieee --handle-signals=yes --warn-overwrite=no --compile=yes --depwarn=yes --cpu-target=native --track-allocation=none --sysimage-native-code=yes '--sysimage=C:\Users\RTX2080\AppData\Local\Julia-1.1.0\lib\julia\sys.dll' -g1 --compiled-modules=yes --optimize=2 'C:\Users\RTX2080\.julia\dev\PackageCompiler\sysimg\run_julia_code.jl'`, ProcessExited(1)) [1]
Stacktrace:
 [1] error(::String, ::Base.Process, ::String, ::Int64, ::String) at .\error.jl:42
 [2] pipeline_error at .\process.jl:785 [inlined]
 [3] #run#515(::Bool, ::Function, ::Cmd) at .\process.jl:726
 [4] run at .\process.jl:724 [inlined]
 [5] #run_julia#1 at C:\Users\RTX2080\.julia\dev\PackageCompiler\src\compiler_flags.jl:225 [inlined]
 [6] #run_julia at .\none:0 [inlined]
 [7] #compile_incremental#62(::Bool, ::Bool, ::Bool, ::Nothing, ::Function, ::String, ::String) at C:\Users\RTX2080\.julia\dev\PackageCompiler\src\incremental.jl:82
 [8] (::getfield(PackageCompiler, Symbol("#kw##compile_incremental")))(::NamedTuple{(:force,),Tuple{Bool}}, ::typeof(compile_incremental), ::String, ::String) at .\none:0
 [9] top-level scope at util.jl:156
```

**Comment by terasakisatoshi** [comment]
your error message contains `could not load library "libGR.dll"` this means you should add `path/to/libGR.dll` into PATH (environment variable). Note that Plots.jl depends on GR.jl

[my gist](https://gist.github.com/terasakisatoshi/14f8fa8bab35683061306f96b1fcf96f) will may help you.

**Comment by KristofferC** [comment]
Fixed in Julia 1.3.1
